### DISCUSSION TITLE
Allow direct access to mapped data in Cell

### DISCUSSION THREAD

**Discussion by lmosele** [TanStack/table#2164] [General]
In an in-house version of React-Table UI we made use of the entire mapped array of the table, even while only using the singular accessor for sorting and filtering, it'd be nice to get access to that in V7 without having to destructure deeply into ` Cell: ({ row: { original } }) => `

Example of current format:
```
 {
                Header: "Last seen",
                accessor: "status",
                Cell: ({ row: { original } }) => {
                  const date = moment(original.lastSeenTimestamp).format(
                    "MMM D, YYYY"
                  );
                  return <Tags date={date} />;
                }
      },
```

Nice to have:
```
 {
                Header: "Last seen",
                accessor: "status",
                Cell: ({ data }) => {
                  const date = moment(data.lastSeenTimestamp).format(
                    "MMM D, YYYY"
                  );
                  return <Tags date={date} />;
                }
      },
```

Not a huge ask but it would clean up our code dramatically!

**Comment by dkulakov** [comment]
Hey, I use accessor instead of Cell

      {
        id: 'licenseNumber',
        isVisible: isTabletUp,
        Header: () => `${licensePageCount} licenses`,
        accessor: row => `${row.licenseNumber} licenses`,
      }

**Reply by lmosele** [reply]
Hm, I feel like this would break sorting though, no?

**Reply by dkulakov** [reply]
It works at the example
https://codesandbox.io/s/github/tannerlinsley/react-table/tree/master/examples/pivoting?file=/src/App.js

**Reply by dkulakov** [reply]
Regarding documentation, it shouldn't break any functionality
https://react-table.js.org/api/useTable#column-options

![image](https://user-images.githubusercontent.com/6547279/79596849-f1c24f00-80e9-11ea-927f-cf00342f4ba7.png)
